### PR TITLE
feat(taskstable): add gpu to task table

### PR DESCRIPTION
**Pull request by bstavroulakis**
Add GPU to task table.

Closes DCOS-21933

To test properly first run a cluster with GPU support. And then create a task that uses GPU.

https://github.com/dcos/dcos-launch/blob/master/dcos_launch/sample_configs/aws-terraform-with-gpus.yaml#L18

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: src/js/stores/MesosStateStore.js [hunk 1]
```diff
@@ -81,6 +81,7 @@ class MesosStateStore extends GetSetBaseStore {
       .distinctUntilChanged()
       .map(message => parsers(this.getLastMesosState(), JSON.parse(message)))
       .do(state => {
+        // The tasks received here have not GPU
```
Comment (bstavroulakis) [inline src/js/stores/MesosStateStore.js]: @nLight For some reason the tasks coming form Mesos don't have GPU, but only CPU and Mem. Is this something we can fix?

Comment (nLight) [reply src/js/stores/MesosStateStore.js]: From the UI point of view if Mesos provides the info it should be there. Can we make sure that mesos provides the info? Unless it does we need an urgent ticket for the Mesos team 💃

**Comment by ahoskins** [comment]
@bstavroulakis What's the status of this? We need to clarify something with mesos or this good?

**Comment by bstavroulakis** [comment]
I'll probably have to create a Mesos ticket

image

File: src/js/stores/MesosStateStore.js (hunk 1 above)
Comment (nLight) [reply src/js/stores/MesosStateStore.js]: Could you please remove the comment?

**Comment by DanielMSchmidt** [comment]
@bstavroulakis You set up a cluster with GPU support, right? Could you share how to do this so that I can review this?

**Comment by nLight** [comment]
Locally those tests pass 🤔

**Comment by DanielMSchmidt** [comment]
@nLight In CI now, too 😀

**Comment by nLight** [comment]
And right align the value please.

**Comment by nLight** [comment]
Also this. Even on full screen MBP 15" I can't see the column if the Sidebar is open. I think we should tweak those columns once again.

screen shot 2018-04-26 at 13 55 59

**Comment by bstavroulakis** [comment]
@nLight Looks like a commit was removed that broke the CSS and the null check. That is fixed now.

**Comment by bstavroulakis** [comment]
@leemunroe What do you believe we should do with Dima's concern? In my screen it looks ok.
@nLight Can you please check with the CSS updates? Now the GPU has a solid width and it will always be visible

image

**Comment by bstavroulakis** [comment]
tasksresponsive

**Comment by bstavroulakis** [comment]
@leemunroe @nLight ^If we are OK with this UI then we are merge this PR and then discuss about the ID column again. 

It will be a long conversation again and I don't want it to keep this PR open forever.

**Comment by bstavroulakis** [comment]
Task columns widths 

taskcolumnsizes

**Comment by DanielMSchmidt** [comment]
@bstavroulakis @leemunroe Can we move the responsiveness discussion into a JIRA ticket to keep track of it and link it here or doesn't that make any sense?

**Comment by juliangieseke** [comment]
Ping @nLight

**Pull request merged**